src/tests/MutatusE2ESpec.scala

```diff
@@ -0,0 +1,237 @@
+package mutatus.tests
+
+import mutatus._
+import probably._
+import com.google.cloud.datastore._
+
+case class MutatusE2ESpec()(implicit runner: Runner) {
```

Comment:
Haha, you've invoked one of my pet-hates here! Don't ever use `2` to mean `to`! ;) I'd rename it to `EndToEndSpec`. (`mutatus` is already in the package name.)